src/redux/books/books.js

```diff
@@ -0,0 +1,30 @@
+// action types
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOk';
```

Comment:
- Defines ADD_BOOK and REMOVE_BOOK as action types.
  - The value of each action type needs to follow the [duck's pattern](https://github.com/erikras/ducks-modular-redux) **app/reducer/ACTION_TYPE**